### DISCUSSION TITLE
Bypassing Mac M2 HDMI 48kHz limitation, question

### DISCUSSION THREAD

**Discussion by Paveldeg** [ExistentialAudio/BlackHole#773] [Q&A] [answered]
I created an aggregate device on my Mac mini m2, which has my Denon AVR and BlackHole. I set sample rate for both to 192kHz. 
I do see 192kHz rate on my AVR when playing hi-res music using VLC player via the aggregate device. 
But I'm curios what this setup does? Does BlackHole app actually pass original 192kHz audio content to my AVR, or just upsamples Mac's internal DAC 48kHz output to 192kHz?

**Comment by devinroth** [comment, marked as answer]
BlackHole doesn’t do any upsampling or any altering of the audio with the exception of applying volume.


Devin
